ExampleMod/build.txt

```diff
@@ -1,5 +1,6 @@
 author = bluemagic123
 version = 0.9.1
+languageVersion = 6
```

Comment:
Well, we've been avoiding using c# 6 code in ExampleMod, not sure if it's a conscious decision or not, but new syntax could be confusing to new modders.